src/components/pages/TrophyRoom/RenderTrophyRoom.js

```diff
@@ -43,28 +53,48 @@ const RenderTrophyRoom = props => {
           </h2>
           <Modal
             title="Achievements"
-            visible={isModalVisible}
-            onOk={handleOk}
-            onCancel={handleCancel}
+            visible={isAchievementModalVisible}
+            onOk={handleAchievmentOk}
+            onCancel={handleAchievementCancel}
+            width={900}
           >
-            <p>Achievement 1</p>
-            <p>Some contents...</p>
-            <p>Some contents...</p>
-            {/* Map over achievement state here */}
+            <div style={{ overflow: 'scroll' }}>
+              <h3>Completed (Way to go!):</h3>
+              {/* {achievementState.map(achievement => {
```

Comment:
If this is your last pr, it might be worth leaving a comment describing the problem you were having making this work. Even if it was just "waiting for backend."
...though I thought the mapping was working. What's the problem here?